newrelic-python.ini

```diff
@@ -63,13 +63,12 @@ security.enable = security_enabled_flag
 security.mode = RASP
```

Comment:
default mode should be IAST.

Reply:
This should be ok. The Automation script takes care of replacing RASP with IAST